SupClient/src/setupHotkeys.ts

```diff
@@ -1,3 +1,5 @@
+export let currentHelpCallback: Function = null;

```

Comment:
This variable is only used in the file so there is no need to export it.
Also we usually don't explicitly affect it to null.
Just `let currentHelpCallback: Function;` would be fine!